crates/core/asset/src/asset/denom_metadata.rs

```diff
@@ -585,6 +579,94 @@ impl Display for Unit {
     }
 }
 
+pub mod parse {
+    use lazy_static::lazy_static;
+
+    lazy_static! {
+        static ref IBC_RE: regex::Regex = regex::Regex::new(
+            r"^(?<path>transfer/[a-z0-9][a-z0-9\-]{0,63}(?:/transfer/[a-z0-9][a-z0-9\-]{0,63})*)/(?<denom>[^/][A-Za-z0-9/._\-]*)$"
+        ).expect("regex compilation works");
+    }
```

Comment:
nit: would be good to have a comment for the source of this regex, if this is coming from some upstream cometbft source

Reply:
unfortunately the source is me, i had to make it up